src/views/HomeIndex.vue

```diff
@@ -35,12 +35,17 @@ const deleteConfirmation = (nodeUuid) => {
 const handleNodeNotFound = () => {
   router.push({ name: 'node.not-found' });
 };
+
+const handleSaveStore = () => {
+  saveStore();
+};
 </script>
 
 <template>
   <div class="container">
     <div class="side-panel">
       <div class="side-panel-header">
+        <Button icon="pi pi-save" aria-label="Save" severity="secondary" @click="handleSaveStore" />
```

Comment:
then you can just use the link from the computed property on the button like this:

```
        <Button
          icon="pi pi-save"
          aria-label="Save"
          severity="secondary"
          as="a"
          :href="downloadNodesLink"
          :download="`solution-navigator-${new Date().toISOString()}.json`"
        />
```